pysat/utils/_core.py

```diff
@@ -0,0 +1,410 @@
+        pysat.data_dir = path
+        pysat._files = re_load(pysat._files)
+        pysat._instrument = re_load(pysat._instrument)
+    else:
+        raise ValueError('Path %s does not lead to a valid directory.' % path)
+
+
+def computational_form(data):
+    """
+    Input Series of numbers, Series, or DataFrames repackaged
+    for calculation.
+
+    Parameters
+    ----------
+    data : pandas.Series
+        Series of numbers, Series, DataFrames
+
+    Returns
+    -------
+    pandas.Series, DataFrame, or Panel
+        repacked data, aligned by indices, ready for calculation
+    """
+
+    from pysat import DataFrame, Series, datetime, Panel
+
+    if isinstance(data.iloc[0], DataFrame):
+        dslice = Panel.from_dict(dict([(i, data.iloc[i])
+                                       for i in xrange(len(data))]))
+    elif isinstance(data.iloc[0], Series):
+        dslice = DataFrame(data.tolist())
+        dslice.index = data.index
+    else:
+        dslice = data
+    return dslice
+
+
+def load_netcdf4(fnames=None, strict_meta=False, file_format=None,
+                 epoch_name='Epoch', units_label='units',
+                 name_label='long_name', notes_label='notes',
+                 desc_label='desc', plot_label='label', axis_label='axis',
+                 scale_label='scale', min_label='value_min',
+                 max_label='value_max', fill_label='fill'):
+    # unix_time=False, **kwargs):
+    """Load netCDF-3/4 file produced by pysat.
+
+    Parameters
+    ----------
+    fnames : string or array_like of strings (None)
+        filenames to load
+    strict_meta : boolean (False)
+        check if metadata across fnames is the same
+    file_format : string (None)
+        file_format keyword passed to netCDF4 routine
+        NETCDF3_CLASSIC, NETCDF3_64BIT, NETCDF4_CLASSIC, and NETCDF4
+    epoch_name : string ('Epoch')
+    units_label : string ('units')
+        keyword for unit information
+    name_label : string ('long_name')
+        keyword for informative name label
+    notes_label : string ('notes')
+        keyword for file notes
+    desc_label : string ('desc')
+        keyword for data descriptions
+    plot_label : string ('label')
+        keyword for name to use on plot labels
+    axis_label : string ('axis')
+        keyword for axis labels
+    scale_label : string ('scale')
+        keyword for plot scaling
+    min_label : string ('value_min')
+        keyword for minimum in allowable value range
+    max_label : string ('value_max')
+        keyword for maximum in allowable value range
+    fill_label : string ('fill')
+        keyword for fill values
+
+    Returns
+    --------
+    out : pandas.core.frame.DataFrame
+        DataFrame output
+    mdata : pysat._meta.Meta
+        Meta data
+    """
+    import pandas as pds
+    import netCDF4
+    import string
+    import copy
+    import pysat
+    try:
+        basestring
+    except NameError:
+        # print ('setting basestring')
```

Comment:
Remove commented print statement or change to Warning.

Reply:
fixed